src/main/kotlin/com/ajaxproject/warehouse/service/CustomerServiceImpl.kt

```diff
@@ -4,86 +4,89 @@ import com.ajaxproject.warehouse.dto.CustomerCreateDto
 import com.ajaxproject.warehouse.dto.CustomerDataDto
 import com.ajaxproject.warehouse.dto.CustomerDataLiteDto
 import com.ajaxproject.warehouse.dto.CustomerUpdateDto
-import com.ajaxproject.warehouse.entity.Customer
+import com.ajaxproject.warehouse.entity.MongoCustomer
+import com.ajaxproject.warehouse.entity.MongoWaybill
 import com.ajaxproject.warehouse.exception.NotFoundException
-import com.ajaxproject.warehouse.repository.CustomerRepository
-import com.ajaxproject.warehouse.repository.WaybillRepository
+import com.ajaxproject.warehouse.repository.MongoCustomerRepository
+import org.bson.types.ObjectId
 import org.springframework.stereotype.Service
 
 @Service
 class CustomerServiceImpl(
-    val customerRepository: CustomerRepository,
-    val waybillRepository: WaybillRepository
+    val mongoCustomerRepository: MongoCustomerRepository
 ) : CustomerService {
 
-    override fun findAllCustomers(): List<CustomerDataLiteDto> {
-        return customerRepository.findAll()
-            .map { it.mapToLiteDto() }
-    }
+    override fun findAllCustomers(): List<CustomerDataLiteDto> =
+        mongoCustomerRepository.findAll().map { it.mapToLiteDto() }
 
-    override fun findById(id: Int): CustomerDataDto {
-        return customerRepository.findById(id)
-            .orElseThrow { NotFoundException("Customer with id $id not found") }
-            .mapToDataDto()
+    override fun getById(id: String): CustomerDataDto {
+        val mongoCustomer: MongoCustomer = mongoCustomerRepository.findById(ObjectId(id))
+            ?: throw NotFoundException("Customer with id $id not found")
+        return mongoCustomer.mapToDataDto()
     }
 
     override fun createCustomer(createDto: CustomerCreateDto): CustomerDataDto {
-        val customer = customerRepository.save(createDto.mapToEntity())
+        val customer: MongoCustomer =
+            mongoCustomerRepository.createCustomer(createDto.mapToEntity())
         return customer.mapToDataDto()
     }
 
-    override fun updateCustomer(updateDto: CustomerUpdateDto, id: Int): CustomerDataDto {
+    override fun updateCustomer(updateDto: CustomerUpdateDto, id: String): CustomerDataDto {
         require(id == updateDto.id) { "Mapping id is not equal to request body id" }
-        val customer: Customer = customerRepository.findById(id)
-            .orElseThrow { NotFoundException("Customer with id $id not found") }
-        customer.setUpdatedData(updateDto)
-        customerRepository.save(customer)
+        var customer: MongoCustomer = mongoCustomerRepository.findById(ObjectId(id))
```

Comment:
```suggestion
        val customer: MongoCustomer = mongoCustomerRepository.findById(ObjectId(id))
```

please, don't reuse the same variable. It's better to create a new one